docs/DeveloperGuide.md

```diff
@@ -260,6 +276,13 @@ The following sequence diagram shows how a SlotsLeft operation goes through the
 
 ![SlotsLeftSequenceDiagram](images/SlotsLeftSequenceDiagram.png)
 
+Design Consideration:
+
+Aspect: Rationale behind why we chose to implement slots_left command
```

Comment:
should this be "implementation of ... command

Reply:
I think "Rationale behind why we chose to implement slots_left command" grammatically makes sense

Reply:
ah that also makes sense, lemme fix now. thx